generators/client-2/templates/src/main/webapp/system.config.js

```diff
@@ -48,7 +51,6 @@
     ngPackageNames.forEach(setPackageConfig);
 
     var config = {
-        meta: { "ui-router-ng2": { format: "cjs" } },
```

Comment:
this is no longer necessary.  systemjs was incorrectly detecting ui-router-ng2.js as es6 module, not commonjs